src/test/storm/modelchecker/csl/CtmcCslModelCheckerTest.cpp

```diff
@@ -446,4 +446,83 @@ namespace {
         EXPECT_NEAR(0.404043, result[0], 1e-6);
         EXPECT_NEAR(0.595957, result[1], 1e-6);
     }
+
+
+    TYPED_TEST(CtmcCslModelCheckerTest, LtlProbabilitiesEmbedded) {
+#ifdef STORM_HAVE_LTL_MODELCHECKING_SUPPORT
+        std::string formulasString = "P=?  [ X F (!\"down\" U \"fail_sensors\") ]";
+        formulasString += "; P=?  [  (! \"down\" U ( F \"fail_actuators\" U \"fail_io\")) ]";
+        formulasString += "; P=?  [ F \"down\" & X G \"fail_sensors\" ]"; // F ("down" & (X (G "fail_sensors")) )
+        formulasString += "; P<1  [ F \"down\" & X G \"fail_sensors\" ]"; // F ("down" & (X (G "fail_sensors")) )
+        formulasString += "; P>0.9  [ F \"down\" & X G \"fail_sensors\" ]"; // F ("down" & (X (G "fail_sensors")) )
+        formulasString += "; P=?  [ ( X X X X X X !\"down\") ]";
+
+        auto modelFormulas = this->buildModelFormulas(STORM_TEST_RESOURCES_DIR "/ctmc/embedded2.sm", formulasString);
+        auto model = std::move(modelFormulas.first);
+        auto tasks = this->getTasks(modelFormulas.second);
+        EXPECT_EQ(3478ul, model->getNumberOfStates());
+        EXPECT_EQ(14639ul, model->getNumberOfTransitions());
+        ASSERT_EQ(model->getType(), storm::models::ModelType::Ctmc);
+
+        auto checker = this->createModelChecker(model);
+        std::unique_ptr<storm::modelchecker::CheckResult> result;
+        if (TypeParam::engine == CtmcEngine::PrismSparse || TypeParam::engine == CtmcEngine::JaniSparse || TypeParam::engine == CtmcEngine::JitSparse) {
+            result = checker->check(this->env(), tasks[0]);
+
+            EXPECT_NEAR(this->parseNumber("0.93458777106264368"),this->getQuantitativeResultAtInitialState(model, result), this->precision());
```

Comment:
If possible, I would prefer the exact fractions here (although exact is *currently* not tested). For example, this would be `6201111489217/6635130141055`
obtained via

```
storm  --prism resources/examples/testfiles/ctmc/embedded2.sm --prop 'P=?  [ X F (!"down" U "fail_sensors") ]' -pc --exact
```

Reply:
Ok! The results are now compared to exact fractions everywhere.